src/cmake/ie_parallel.cmake

```diff
@@ -169,12 +169,12 @@ macro(ov_find_package_tbb)
                     find_file(HWLOC_DLL NAMES ${hwloc_dll_name} PATHS "${TBB_dir}" DOC "Path to hwloc.dll")
 
                     if(NOT HWLOC_DLL)
-                        message(FATAL_ERROR "Failed to find ${hwloc_dll_name} in ${TBB_dir}")
+                        message(WARNING "Failed to find ${hwloc_dll_name} in ${TBB_dir}")
```

Comment:
yes, now hwloc static library is built.

Reply:
Do you mean that TBB package was changed and we can skip this library for new TBB package? I don't understand who built the hwloc

Reply:
I mean that the hwloc static library is linked when tbbbind is compiled, just like the tbbbind static library was linked when openvino was compiled without tbbbind dynamic library. The tbbbind library is larger than the previous one, and no longer depends on the hwloc dynamic library, because the hwloc library has been compiled into the tbbbind library.

Reply:
It's OK to convert this error into conditional check. I would even remove warning here to avoid confusions from users